src/mumble/NetworkConfig.ui

```diff
@@ -304,14 +304,14 @@ Prevents the client from downloading images embedded into chat messages with the
       <item>
        <widget class="QCheckBox" name="qcbHideOS">
         <property name="toolTip">
-         <string>Prevent OS information being sent to the server</string>
+         <string>Prevent OS information being sent to Mumble and web servers</string>
         </property>
         <property name="whatsThis">
-         <string>&lt;b&gt;Don't send OS information to the server&lt;/b&gt;&lt;br/&gt;
-Prevents the client from sending potentially identifying information about the operating system to the server.</string>
+         <string>&lt;b&gt;Don't send OS information to servers&lt;/b&gt;&lt;br/&gt;
+Prevents the client from sending potentially identifying information about the operating system to the Mumble server and web servers.</string>
         </property>
         <property name="text">
-         <string>Do not send OS information to Mumble servers</string>
+         <string>Do not send OS information to Mumble and web servers</string>
```

Comment:
I think we should be more clear here. Sorry if this is annoying.

In "Do not send OS information to Mumble and web servers", the "to Mumble" part could be interpreted as "to the Mumble project" (or, to people not familiar with Mumble or community-based FOSS projects, "the company behind Mumble").

The straight-forward way to fix it would be:

Do not send OS information to Mumble servers or web servers.

It can definitely fit in the ConfigDialog, even though the string is a bit on the long side.

Maybe we could change it to "Do not send OS information", and explain it in the tooltip?